src/components/author.tsx

```diff
@@ -18,19 +18,24 @@ const img = css`
 `;
 
 type Props = {
+  number: Number;
```

Comment:
number isn't really self-explaining. Maybe order?

Reply:
Or perhaps `orderIndex`.

Also, just use the literal `number` for the type.

```suggestion
  orderIndex: number;
```

Reply:
Changing that to `index`.